packages/ariakit/src/checkbox/__examples__/checkbox-as-button/style.css

```diff
@@ -0,0 +1,17 @@
+@import url("../checkbox/style.css");
+
+.checkbox {
+  @apply
+    w-5
+    h-5
+    rounded
+    flex
+    justify-center
+    items-center
+    bg-primary-1
+    border-primary-1
+    border
+    aria-checked:after:content-["X"]
```

Comment:
`content` isn't compiled nicely with Tailwind, we can just use vanilla CSS in this case.